### PR TITLE
[fix] company naming issue

### PR DESCRIPTION
**Pull request by pratu16x7**
Fixes, but too much lag due update_global_search; checks pending:
charts

### CODE REVIEW

**Comment by pratu16x7** [comment]
Lag fixed via frappe/frappe#3455

charts

**Comment by robert1112** [comment]
I got the same issue as yours. Anyone can help? Thanks.

File: chart_of_accounts_builder/utils.py [hunk 1]
```diff
@@ -77,8 +77,8 @@ def create_company(company_name, country, default_currency, chart_of_accounts, f
 	company.abbr = random_string(3)
 	company.forked = 1
 	company.forked_from = forked_from
-	company = append_number_if_name_exists("Company", company)
-	company.company_name = company.name
+	numbered_company_name = append_number_if_name_exists("Company", company_name)
+	company.company_name = numbered_company_name
```
Comment (nabinhait) [inline chart_of_accounts_builder/utils.py]: I think company.name = numbered_company_name  is also required to set

Comment (robert1112) [reply chart_of_accounts_builder/utils.py]: Hi is it fixed? I dont quite understand. Thanks.

**Pull request merged**